src/sinks/gcp_chronicle/chronicle_unstructured.rs

```diff
@@ -52,6 +52,35 @@ use crate::{
     tls::{TlsConfig, TlsSettings},
 };
 
+
+/// Compression configuration.
+#[configurable_component]
+#[derive(Clone, Debug, Derivative, Eq, PartialEq)]
+#[derivative(Default)]
+#[serde(rename_all = "snake_case")]
+#[configurable(metadata(
+    docs::enum_tag_description = "The compression algorithm to use for sending."
+))]
+pub enum ChronicleCompression {
+    /// No compression.
+    #[derivative(Default)]
+    None,
+
+    /// [Gzip][gzip] compression.
+    ///
+    /// [gzip]: https://www.gzip.org/
+    Gzip,
```

Comment:
Would there be any demand to configure the compression level, given the different performance characteristics that could bring?

Reply:
I can't speak to a required demand but adding configurability doesn't seem to difficult and wouldn't add too much overhead when configuring compression so I've added it in